crypto/ct/ct_log.c

```diff
@@ -182,7 +182,10 @@ static int ctlog_store_load_log(const char *log_name, int log_name_len,
         return 1;
     }
 
-    sk_CTLOG_push(load_ctx->log_store->logs, ct_log);
+    if (!sk_CTLOG_push(load_ctx->log_store->logs, ct_log)) {
+        CTLOG_free(ct_log);
+        return 0;
```

Comment:
The comment before the start of the function suggests we shouldn't return 0 unless "something very bad happens". This probably qualifies because this is an internal error...but perhaps the comment at the top could do with tweaking.